CodeGen/Sources/LucidCodeGen/Meta/MetaEntityGraph.swift

```diff
@@ -336,6 +345,23 @@ struct MetaEntityGraph {
             )
     }
 
+    private func setEndpointResultPayloadFunction() -> Function {
```

Comment:
super nitpick: update the function name to match the change `setEndpointResultMetadataFunction()`